src/components/Synthetics/TradeBox/tradeboxConstants.tsx

```diff
@@ -0,0 +1,50 @@
+import { msg } from "@lingui/macro";
+
+import { TradeMode, TradeType } from "domain/synthetics/trade";
+
+import LongIcon from "img/long.svg?react";
+import ShortIcon from "img/short.svg?react";
+import SwapIcon from "img/swap.svg?react";
+
+import "./TradeBox.scss";
+
+export type Props = {
+  setPendingTxns: (txns: any) => void;
+};
```

Comment:
does not look like a constant, is it used anywhere? because i can see the tradebox still has this type, maybe this one is a mistake ?

Reply:
Correct, I will remove, and scss import as well